lib/shoryuken/manager.rb

```diff
@@ -73,6 +73,8 @@ def dispatch_now
           return
         end
 
+        fire_event(:dispatch, false)
```

Comment:
@Erol `false` is the default value for `reverse`, I guess you can omit it. 

Can you add an expectation in the manager spec for this `fire_event` call?